### PR TITLE
Fix double typo

### PR DESCRIPTION
**Pull request by williamdes**
cc @OrRosenblatt 

I found this by running your action that works very nicely :+1:

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ async function run() {
         if (invalidJsons.length > 0) {
             core.setFailed('Failed to validate all JSON files.');
         } else {
-            core.info(`✅ All files were validate succesfully.`);
+            core.info(`✅ All files where validated succesfully.`);
```
Comment (rpbaltazar) [inline src/main.ts]: Should be `All files were validated successfully`

Comment (kingthorin) [inline src/main.ts]:
```suggestion
            core.info(`✅ All files were validated succesfully.`);
```

Comment (williamdes) [reply src/main.ts]: ??

Comment (williamdes) [reply src/main.ts]: You are maybe right but please post links so I can improve my knowledge

Comment (kingthorin) [reply src/main.ts]: https://www.dictionary.com/browse/where?s=t
https://www.dictionary.com/browse/were?s=t

Comment (williamdes) [reply src/main.ts]: Thank you, fixed

**Pull request merged**